compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -237,25 +237,33 @@ loco::NodeShape use_paddings(const CIRCLENODE *node, const luci::CircleConst *pa
   {
     int32_t idx = ni * 2;
     int value = input_shape.dim(ni).value();
-    if (paddings->dtype() == S32)
+    if (!input_shape.dim(ni).known())
```

Comment:
(optional)

```cxx

  for (int32_t ni = 0; ni < n; ++ni)
  {
    if (not input_shape.dim(ni).known())
    {
      output_shape.dim(ni).unset();
      continue;
    }
    int32_t idx = ni * 2;
```

Reply:
I thought continue statement might violate convention.😂
If it's fine, this is cleaner code.